src/theme/index.css

```diff
@@ -120,19 +122,19 @@ body {
 }
 
 /* Right Sidebar Freemium Container Maybe later Hover Background */
-.fefqyxr:not([data-disabled='true'])[data-hovered='true'] {
-  background: var(--primary-color) !important;
+.fefqyxr:not([data-disabled="true"])[data-hovered="true"] {
+  background: #000000 !important;
 }
 
 /* Right Sidebar Hide Assistant Background */
 ._78e3fb69-sidebar-overviewBtnA11yContrast {
-  background: var(--primary-color) !important;
+  background-color: #1f212a !important;
```

Comment:
If this color is more accurate than the `--primary-color`, then please add it to the `:root` on the top of the file.